plugins/modules/slack.py

```diff
@@ -129,6 +129,15 @@
     type: list
     elements: dict
     version_added: 1.0.0
+  prepend_hash:
```

Comment:
`prepend_hash` was the first thing that came up in my mind, if you have a better idea, please use that :)

Reply:
I can't think of a better name :-)